plugins/call_button/call_button.scss

```diff
@@ -0,0 +1,140 @@
+
+    &:hover {
+        background: rgba(var(--button-bg-rgb), 0.08);
+        color: rgba(var(--button-bg-rgb), 1);
+        fill: rgba(var(--button-bg-rgb), 1);
+    }
+
+    &:active {
+        background: rgba(var(--button-bg-rgb), 0.16);
+        color: rgba(var(--button-bg-rgb), 1);
+        fill: rgba(var(--button-bg-rgb), 1);
+    }
+
+    i,
+    span {
+        font-size: 14px;
+        line-height: 14px;
+    }
+
+    i::before,
+    span::before {
+        margin: 0;
+    }
+
+    svg {
+        width: 14px;
+        height: 14px;
+    }
+
+    span.call-button-label {
+        padding: 1px 0;
+        margin: 0 6px;
+        font-size: 12px;
+        font-weight: 600;
+        line-height: 9px;
+    }
+}
+
+button.call-button.dropdown {
+    color: rgba(var(--center-channel-color-rgb), 0.56);
+    fill: rgba(var(--center-channel-color-rgb), 0.56);
+}
+
+button.call-button.active {
+    background: rgba(var(--button-bg-rgb), 0.08);
+    color: rgba(var(--button-bg-rgb), 1);
+    fill: rgba(var(--button-bg-rgb), 1);
+}
+
+button.call-button.disabled {
+    color: rgba(var(--center-channel-color-rgb), 0.32);
+    fill: rgba(var(--center-channel-color-rgb), 0.32);
+
+    &:hover {
+        background: inherit;
+        color: rgba(var(--center-channel-color-rgb), 0.32);
+        fill: rgba(var(--center-channel-color-rgb), 0.32);
+    }
+
+    &:active {
+        background: revert;
+        color: rgba(var(--center-channel-color-rgb), 0.32);
+        fill: rgba(var(--center-channel-color-rgb), 0.32);
+    }
+}
+
+.Menu .MenuItem > button.call-button-dropdown {
+    display: flex;
+    padding: 6px 20px 6px 16px;
+    margin: 0;
+    font-size: 14px;
+    line-height: 20px;
+
+    svg {
+        width: 16px;
+        height: 16px;
+        align-self: flex-start;
+        margin-right: 12px;
+        margin-left: 4px;
+        fill: rgba(var(--button-bg-rgb), 1);
+    }
+
+    i {
+        align-self: flex-start;
+        margin-right: 12px;
+        margin-left: 4px;
+        color: rgba(var(--button-bg-rgb), 1);
+        font-size: 16px;
+    }
+
+    i::before {
+        margin: 0;
+    }
+
+    .call-button-dropdown-sublabel {
+        margin-top: 4px;
+        color: rgba(var(--center-channel-color-rgb), 0.56);
+        font-size: 12px;
+        line-height: 16px;
+    }
+
+    div {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.Menu .MenuItem > button.call-button-dropdown.disabled {
+    color: rgba(var(--center-channel-color-rgb), 0.32);
+    fill: rgba(var(--center-channel-color-rgb), 0.32);
+
+    svg {
+        width: 16px;
+        height: 16px;
+        fill: rgba(var(--center-channel-color-rgb), 0.32);
+    }
+
+    i {
+        color: rgba(var(--center-channel-color-rgb), 0.32);
+        font-size: 16px;
+    }
+
+    i::before {
+        margin: 0;
+    }
+
+    .call-button-dropdown-sublabel {
+        color: rgba(var(--center-channel-color-rgb), 0.32);
+        font-size: 12px;
+    }
+}
```

Comment:
@streamer45 Sorry for the massive suggestion, but having them all separated would be even less comprehensible. If you have questions about these proposed changes feel free to reach out.

```suggestion
button.call-button {
    display: flex;
    height: 32px;
    place-items: center;
    place-content: center;
    padding: 10px 12px;
    border-radius: 4px;
    color: rgb(var(--button-bg-rgb));
    fill: currentColor;

    &.dropdown {
        color: rgba(var(--center-channel-color-rgb), 0.56);
        fill: currentColor;
    }
    
    &.active {
        background: rgba(var(--button-bg-rgb), 0.08);
        color: rgb(var(--button-bg-rgb));
        fill: currentColor;
    }

    &:hover {
        background: rgba(var(--button-bg-rgb), 0.08);
        color: rgb(var(--button-bg-rgb));
        fill: currentColor;
    }

    &:active {
        background: rgba(var(--button-bg-rgb), 0.16);
        color: rgb(var(--button-bg-rgb));
        fill: currentColor;
    }
    
    &.disabled {
        color: rgba(var(--center-channel-color-rgb), 0.32);
        fill: currentColor;
    
        &:hover {
            background: inherit;
            color: inherit;
            fill: currentColor;
        }
    
        &:active {
            background: revert;
            color: inherit;
            fill: currentColor;
        }
    }

    i,
    span {
        font-size: 14px;
        line-height: 14px;
    }

    i::before,
    span::before {
        margin: 0;
    }

    svg {
        width: 14px;
        height: 14px;
    }

    .call-button-label {
        padding: 1px 0;
        margin: 0 6px;
        font-size: 12px;
        font-weight: 600;
        line-height: 9px;
    }
}

.Menu .MenuItem > button.call-button-dropdown {
    display: flex;
    padding: 6px 20px 6px 16px;
    margin: 0;
    font-size: 14px;
    line-height: 20px;

    i,
    svg {
        align-self: flex-start;
        margin-right: 12px;
        margin-left: 4px;
    }

    svg {
        width: 16px;
        height: 16px;
        fill: rgb(var(--button-bg-rgb));
    }

    i {
        color: rgb(var(--button-bg-rgb));
        font-size: 16px;

        &::before {
            margin: 0;
        }
    }

    .call-button-dropdown-sublabel {
        margin-top: 4px;
        color: rgba(var(--center-channel-color-rgb), 0.56);
        font-size: 12px;
        line-height: 16px;
    }

    div {
        display: flex;
        flex-direction: column;
    }
    
    &.disabled {
        color: rgba(var(--center-channel-color-rgb), 0.32);
        fill: rgba(var(--center-channel-color-rgb), 0.32);
        
        svg {
            fill: rgba(var(--center-channel-color-rgb), 0.32);
        }
        
        i {
            color: rgba(var(--center-channel-color-rgb), 0.32);
        
            &::before {
                margin: 0;
            }
        }
        
        .call-button-dropdown-sublabel {
            color: rgba(var(--center-channel-color-rgb), 0.32);
            font-size: 12px;
        }
    }
}

```

Reply:
Somehow this is not working as expected:

```css
    &.disabled {
        color: rgba(var(--center-channel-color-rgb), 0.32);
        fill: currentColor;
    
        &:hover {
            background: inherit;
            color: inherit;
            fill: currentColor;
        }
    
        &:active {
            background: revert;
            color: inherit;
            fill: currentColor;
        }
    }
```

When hovering the disabled button, the color as defined above (`rgba(var(--center-channel-color-rgb), 0.32)`) is not retained.

Reply:
I might need to check out the PR and test it locally then ... let me do that tomorrow (or maybe later today) and get back here